integration-tests/manifests/cloudwatch-agent-daemonset.yaml

```diff
@@ -0,0 +1,80 @@
+apiVersion: cloudwatch.aws.amazon.com/v1alpha1
```

Comment:
is there a reason we are not using the template in the helm chart? https://github.com/aws-observability/helm-charts/blob/main/charts/amazon-cloudwatch-observability/templates/linux/cloudwatch-agent-daemonset.yaml

Reply:
I think we can use that template, but it would require passing in a lot of values, for which it seemed more frugal to just have set up already.